src/cmd/build.js

```diff
@@ -110,7 +111,9 @@ async function defaultPackageCreator(
 
   await streamToPromise(stream);
 
-  log.info(`Your web extension is ready: ${extensionPath}`);
+  if (showReadyMessage) {
```

Comment:
Hehe, no worries, managing git branches can be tricky sometimes and git commands/workflows are not always very intuitive :-)
(and after a bit of practice everything looks simpler than it really is ;-))

Sometimes it is simpler to rename the old branch (`git branch -m OLDNAME NEWNAME`), switch to a new pristine branch, named like the one pushed on github (`git checkout master && git checkout -b OLDNAME`), and start again (and maybe cherry-picking what is still useful from the previous branch)

But in my opinion, there is a lot of value also into learning how to rewrite the existent history in the branch that we are working on (so that we feel confortable that we can manage every situation that we can find ourselves while we are working on a git branch), and so I'd like to give you more information of the topic :-D

There are many ways to get rid of commits from a branch in git, my favorite one is the interactive rebase, e.g. we can ask git to let us rebase interactively the last 4 commits using the command:

```
$ git rebase -i HEAD~4
```

and it will open our configured editor (the one in the EDITOR environment variable or the standard one of our system) with the
list of the commits that we are going to rebase:

```
pick 062d62a fix: Don't log info about the built extension when signing
pick 10f8b7b fix: the code consistency for issue 181
pick 7edc47f fix: another code inconsistency for issue 181
pick 77626b8 fix: issue 88 first commit
```

In the opened editor, if you remove the first three lines, save and exit, git will remove the correspondent commits from the branch you are working on.

The following are a couple of additional resources on the topic that you may find interesting/useful:

- http://learngitbranching.js.org/ has a nice interactive tutorial on both "reverting single commit" and "interactive rebasing", with nice diagrams to help you visualizing what is happening in the git repo
- https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase has a short description and a diagram
- https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History has more detailed informations on how the git interactive rebase works